### ISSUE TITLE
Suggestions does not show placeholders for static methods

### ISSUE THREAD

**Issue by Cassandra-Clark** [enso-org/enso#8866]
### What type of issue is this?

Permanent – Occurring repeatably

### Is this issue blocking you from using Enso?

- [ ] Yes, I can't use Enso because of this issue.

### Is this a regression?

- [ ] Yes, previous version of Enso did not have this issue.

### What issue are you facing?

When using a static method in the GUI, the method's placeholders are not displayed.

### Expected behaviour

static methods should have placeholders displayed

### How we can reproduce it?

attempt to use a static method (eg Google_Api.initialize)
no arguments are populated

### Screenshots or screencasts

![image](https://github.com/enso-org/enso/assets/40743109/7ea0f6bc-eebb-4146-aa4e-1a509ddfb8f9)

### Enso Version

enso-win-2024.1.1-nightly.2024.1.4

### Browser or standalone distribution

Standalone distribution (local project)

### Browser Version or standalone distribution

standalone

### Operating System

Windows

**Comment by farmaazon** [comment]
This is an issue of only some of the static methods, because `Data.read` looks just fine. Google_Api is a library excluded from stdlib by default - may it affect the suggestion database somehow @hubertp @4e6  ?

**Comment by 4e6** [comment]
Yes, `Google_Api` is not included in the standard library edition. And I suppose that the suggestions for it are not loaded on startup when the engine loads the standard library.

**Comment by 4e6** [comment]
I'm not sure about the status of this library. It was not maintained for a long time. I suppose that just including it in the edition should fix this issue.

**Comment by farmaazon** [comment]
Thanks. I think I saw a PR which brings it back - so when it lands, I will check if we have parameters.

**Comment by farmaazon** [comment]
Using anything from Google_Api now is just fine.
